app/resources/api/v2/tube_purpose_resource.rb

```diff
@@ -2,30 +2,39 @@
 
 module Api
   module V2
-    # Provides a JSON API representation of a tube purpose.
-    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    #
+    # @note Access this resource via the `/api/v2/tube_purposes/` endpoint.
+    #
+    # Provides a JSON:API representation of {Tube::Purpose}.
+    #
+    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package for Sequencescape's implementation
+    # of the JSON:API standard.
     class TubePurposeResource < BaseResource
-      model_name 'Tube::Purpose'
+      model_name 'Tube::Purpose', model_hint: true
```

Comment:
Is this model_hint correct? should this be add_model_hint?

Reply:
Again I'm not sure.  I have recollections of it not working as an endpoint without me adding this, but I honestly am not sure what the differences are between all these model names and model hints.  I have read the documentation, and it doesn't explain it well enough.

Reply:
Thanks, so I tried this out this morning and it doesn't need the model hint adding explicitly (and this isn't the right way to add it).  It works without.  Not sure why I felt the need to add it, but I've removed this again.